src/intrinsics/fb-www/fb-mocks.js

```diff
@@ -234,3 +234,108 @@ export function createFbMocks(realm: Realm, global: ObjectValue | AbstractObject
     createMagicGlobalObject(realm, global, objectName);
   }
 }
+
+export function createMockTimeSlice(realm: Realm, timeSliceRequireName: string): ObjectValue {
```

Comment:
Can you explain more about which patterns make this necessary?

Reply:
I dig a lot of digging and found TimeSlice needed to be mocked, like React, otherwise it havocs far too much on the init path making it impossible to render the root UFI components.